content/docs/getting-started.md

```diff
@@ -21,18 +21,24 @@ If you need to install Go or update your currently installed one, please refer t
 
 You can download the latest source build for containerd from the [Downloads](/downloads) page and then use your favorite process supervisor to get the daemon started.
 
-To install version {{< latest >}}, run the following commands to download and unzip the source code on the target machine.
+### Systemd-based distributions
+To install version {{< latest >}} on a distribution with Systemd, run the following commands:
 
 ```shell
-wget https://github.com/containerd/containerd/archive/v{{< latest >}}.zip
-unzip v{{< latest >}}.zip
+wget https://github.com/containerd/containerd/releases/download/v{{< latest >}}/cri-containerd-cni-{{< latest >}}-linux-amd64.tar.gz
+sudo tar --no-overwrite-dir -C / -xzf cri-containerd-cni-${VERSION}-linux-amd64.tar.gz
```

Comment:
sorry--should have caught this the first time; but given the `{{< latest >}}` incantation can put the right value here for `${VERSION}` (which won't do anything unless the user sets it), why don't we replace both instances of `${VERSION}` with the replace handler? Should be good to merge after that. Thanks!